hw/xbox/nv2a/nv2a.c

```diff
@@ -410,6 +410,8 @@ static void nv2a_realize(PCIDevice *dev, Error **errp)
 
     d = NV2A_DEVICE(dev);
 
+    pci_set_word(dev->config + PCI_SUBSYSTEM_VENDOR_ID, 0);
+    pci_set_word(dev->config + PCI_SUBSYSTEM_ID, 0);
```

Comment:
Possibly a comment that setting them in `nv2a_class_init` doesn't seem to be enough.

Reply:
Review addessed, improved commit message, added comments. :ok_hand: